index.js

```diff
@@ -734,25 +729,30 @@
   //. See also [`reject`](#reject).
   //.
   //. ```javascript
-  //. > S.filter(S.odd, [1, 2, 3])
+  //. > S.filter (S.odd) ([1, 2, 3])
   //. [1, 3]
   //.
-  //. > S.filter(S.odd, {x: 1, y: 2, z: 3})
+  //. > S.filter (S.odd) ({x: 1, y: 2, z: 3})
   //. {x: 1, z: 3}
   //.
-  //. > S.filter(S.odd, S.Nothing)
+  //. > S.filter (S.odd) (S.Nothing)
   //. Nothing
   //.
-  //. > S.filter(S.odd, S.Just(0))
+  //. > S.filter (S.odd) (S.Just (0))
   //. Nothing
   //.
-  //. > S.filter(S.odd, S.Just(1))
-  //. Just(1)
+  //. > S.filter (S.odd) (S.Just (1))
+  //. Just (1)
   //. ```
+  function filter(pred) {
+    return function(filterable) {
+      return Z.filter (pred, filterable);
+    };
+  }
```

Comment:
We are defining `filter` rather than setting `impl` to `curry2 (Z.filter)` in order to partially apply the function in other definitions.